pkg/config/app_config.go

```diff
@@ -250,6 +250,72 @@ type StatsConfig struct {
 	MaxDuration time.Duration `yaml:"maxDuration,omitempty"`
 }
 
+type KeyConfig struct {
+	Quit            []string
+	ScrollUpMain    []string
+	ScrollDownMain  []string
+	AutoScrollMain  []string
+	ShowOptionsMenu []string
+	CustomCommand   []string
+	Project         struct {
+		EditConfig      []string
+		OpenConfig      []string
+		PreviousContext []string
+		NextContext     []string
```

Comment:
I think cross-cutting keybindings like NextContext and PreviousContext should sit at the top level rather than under each panel that uses them. That way the user only needs to override the key in one place.

Reply:
Agreed, as I copied everything from the keybindings and i found a lot of repeating configs, will look more into that